composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ibexa/core": "^4.0",
+        "ibexa/content-forms": "*",
```

Comment:
We should match minor version of `ibexa/core` require here.

Reply:
I like to keep in it always in sync, purely to limit somehow installing older versions of some packages on later versions of Ibexa to reduce chances having issues, if, by chance, `ibexa/core` does not limit it. And it's more clear which versions of Ibexa packages we support.

Besides, all `ibexa/*` packages keep minor version the same for all packages within a single minor product release anyhow.

Reply:
OK, updated in https://github.com/netgen/site-bundle/pull/34/commits/6eb651d1a4e953951cbba7e933f357e64e36af70.